frontend/src/app/features/auth/reset-password/reset-password.component.ts

```diff
@@ -0,0 +1,67 @@
+import * as loginPageActions from '@store/actions/login-page.actions';
+
+import { ActivatedRoute, Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { getPasswordValidators, hasLengthErrors, passwordMatchingValidator } from '@utils/validators/user-validators.helper';
+
+import { AuthService } from '@api/auth.service';
+import { BaseComponent } from '@shared/components/base/base.component';
+import { Store } from '@ngrx/store';
+
+@Component({
+    selector: 'app-reset-password',
```

Comment:
Also, 'app-' is a bad prefix. It is default. Better have something unique according to your app name - e.g. 'so-'

Reply:
Fixed